Ionic_Example/package.json

```diff
@@ -0,0 +1,82 @@
+{
+  "name": "Ionic_Example",
```

Comment:
Change name if we modify the folder structure, "ionic-angular-example" or something.